setup.py

```diff
@@ -17,7 +17,7 @@
 
 setup(
     name='python-dependency-verifier',
-    version='0.1',
+    version='0.2',
```

Comment:
I'm not sure I understand the problem.
This is a substantial change. no ? Or a Major release ?